modules/storages/spec/factories/storage_factory.rb

```diff
@@ -47,18 +47,9 @@
     trait :as_generic do
       provider_type { 'Storages::Storage' }
     end
-  end
-
-  factory :nextcloud_storage,
-          parent: :storage,
-          class: '::Storages::NextcloudStorage' do
-    provider_type { Storages::Storage::PROVIDER_TYPE_NEXTCLOUD }
-    sequence(:host) { |n| "https://host#{n}.example.com" }
 
     trait :as_automatically_managed do
       automatically_managed { true }
-      username { 'OpenProject' }
```

Comment:
Hi @akabiru, I removed `username` and `password` from the storage factory, as I thought it is not needed anymore. Please correct me if I am wrong.